tests/integration/components/cf-field/input/powerselect-test.js

```diff
@@ -0,0 +1,136 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "ember-qunit";
+import { click, render } from "@ember/test-helpers";
+import hbs from "htmlbars-inline-precompile";
+
+module("Integration | Component | cf-field/input/powerselect", function(hooks) {
+  setupRenderingTest(hooks);
+
+  test("it renders (single)", async function(assert) {
+    assert.expect(1);
+
+    this.set("field", {
+      id: "test",
+      answer: {
+        _valueKey: "stringValue",
+        stringValue: "option-3"
+      },
+      question: {
+        __typename: "ChoiceQuestion",
+        choiceOptions: {
+          edges: [
+            { node: { slug: "option-1", label: "Option 1" } },
+            { node: { slug: "option-2", label: "Option 2" } },
+            { node: { slug: "option-3", label: "Option 3" } }
+          ]
+        }
+      }
+    });
```

Comment:
maybe better to set `singleChoiceField` and `multipleChoiceField` in beforeEach to avoid repeating the lengthy `set` call in each test?

Reply:
Good idea!